internal/network/manager.go

```diff
@@ -159,6 +159,22 @@ func (m *Manager) SendRTP(packet *rtp.Packet) {
 	}
 }
 
+// SendRTCP finds a connected port and sends the passed RTCP packet
+func (m *Manager) SendRTCP(pkt []byte) {
+	m.portsLock.Lock()
```

Comment:
I noticed in several places we lock around things other than ports -- should we rename portsLock to something else? Or reduce the scope of the locks?

Reply:
It still is just for the ports (iceAgent interaction is thread safe)

I moved the locks down, should read a bit nicer!